HW03_Xingruo_Summer_Zhang.Rmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "HW03_Xingruo_Summer_Zhang"
+author: "Summer Zhang"
+date: "7/20/2020"
+output: md_document
+---
+
+```{r}
+# Load Packages
+library(ggplot2)
+library(readr)
+library(magrittr)
+library(tidyr)
+library(dplyr)
+
+# Read Data
+us <- read_csv("us.csv")
+states <- read_csv("us-states.csv")
+```
+
+```{r}
+head(states)
+```
+
+## Graph 1: How Caes in the Entire U.S. Grew
+```{r}
+# Graph the cases
+ggplot(us, aes(x = date, y = cases, linetype = "Cases", color = "Cases")) +
+  geom_line(size = 1.5) +
+  # Add a layer of deaths accordingly
+  geom_line(aes(x = date, y = deaths, linetype = "Deaths", color = "Deaths"), size = 1.5) +
+  # solid for cases, dashed for deaths
+  scale_linetype_manual(values = c("solid", "dotted")) + 
+  scale_color_manual(values = c('skyblue', 'tomato')) +
+  labs(color  = "", linetype = "", title = "COVID-19 Cases and Deaths in the U.S.") +
+  # Bolden and center the title
+  theme(plot.title = element_text(face = "bold", hjust = 0.5)) +
+  # Change the theme
+  theme_classic()
+```
+
+## Graph 2: How Cases and Deaths in Those States with Most Cases Grew
+```{r}
+# Select the 3 states that have the most cases up until 07/17
+# Order the states by cases on 07/17
+ordered_states <- states %>% 
```

Comment:
I have been looking for a way to subset the same dataset but couldn't figure out, learned it from yours, thanks!